README.md

```diff
@@ -29,102 +39,26 @@ This project has 2 parts: the Docker environment and a set of tools for theme de
 ```
 # copy the files
 git clone https://github.com/visiblevc/wordpress-starter.git
-rm -rf .git Dockerfile run.sh README.md CHANGELOG.md ISSUE_TEMPLATE.md
+
+# navigate to example directory
+cd wordpress-starter/example
 
 # start the website at localhost:8080
-docker-compose up
+docker-compose up -d && docker-compose logs -f wordpress
```

Comment:
That's how I start it. It allows you to watch build logs and know when exactly the build is finished, rather than just trying localhost:8080 a bunch of times until it works.